03_launch_mgmt_cluster.sh

```diff
@@ -25,6 +25,11 @@ export GOPATH
 # BMO_RUN_LOCAL : run the baremetal operator locally (not in Kubernetes cluster)
 # CAPM3_RUN_LOCAL : run the CAPI operator locally
 
+export DEPLOY_KERNEL_URL="http://172.22.0.1:6180/images/ironic-python-agent.kernel"
+export DEPLOY_RAMDISK_URL="http://172.22.0.1:6180/images/ironic-python-agent.initramfs"
+export IRONIC_URL="http://172.22.0.1:6385/v1/"
+export IRONIC_INSPECTOR_URL="http://172.22.0.1:5050/v1/"
```

Comment:
Why not parameterizing the IP?

Reply:
yes, you cannot rely on this being the IP, for example with IPv6 deployments, or when the provisioning network has a different CIDR. There is already a variable containing this info, check lib/network.sh or something like that

Reply:
Parameterized, can you please check @kashifest @maelk